### ISSUE TITLE
Falling Kerbals

### ISSUE THREAD

**Issue by DavidShade** [Keniamin/KerbinSideGAP#2]
The Kerbals I'm picking up for contracts are spawning up in the air. Specifically at Dundard's Edge they are spawning at such a high altitude that some die on impact.  I'm including my save file.
[persistent.zip](https://github.com/Keniamin/KerbinSideGAP/files/1083256/persistent.zip)

**Comment by Keniamin** [comment]
Yes, known issue. Unfortunately, not too much ideas. I tried to fix it but it looks like a bug in a Contract Configurator behaviors, or even in the internal KSP code.

I thought the source of the problem can be in conversions between altitude above sea level and above the surface. But when I tried to use surface-related one my Kerbals, on the contrary, spawned under the surface. You gave me another proof that I'm setting it correctly: in your save if you'll launch your vessel at Dundard's Edge helipad (instead of runway) you'll see Kerbals where they must be, just above the surface (screenshot 1). But when you go closer (~200 meters) they "jump" to sky (screenshot 2). So it is not about contract parameters, it is about how Kerbals loading is processed by the game.

I've created a related issue in the Contract Configurator repo, may be its author can clarify my suppositions. Attaching a log for better debug: [KSP-strange-physics.zip](https://github.com/Keniamin/KerbinSideGAP/files/1089749/KSP-strange-physics.zip)

![screenshot1](https://user-images.githubusercontent.com/10873738/27357103-88f47e4c-5619-11e7-88d5-7e5f56b15fac.png)
![screenshot2](https://user-images.githubusercontent.com/10873738/27357102-88f3f972-5619-11e7-83b7-a7e107587f0e.png)

**Comment by jrossignol** [comment]
Confirmed as a Contract Configurator issue.  Will be fixed in 1.23.2.
